src/components/Card/Card.tsx

```diff
@@ -39,15 +51,29 @@ export const Card: React.FC = () => {
       </div>
 
       <div className="card__buy">
-        <a href="/" className="card__buy--add">
-          {' '}
-          Add to cart
-          {' '}
-        </a>
-
-        <a href="/" className="card__buy--heart">
-          <img src={heart} alt="heart_icon" className="card__icon" />
-        </a>
+        <Link
+          to="/purchases"
+          className={classNames('card__buy--add', {
+            'card__buy--add-active': isActiveToCard,
+          })}
+          onClick={handleCardButton}
+        >
+          {!isActiveToCard ? 'Add to cart' : 'Added'}
+        </Link>
+
+        <Link
+          to="/favourites"
+          className={classNames('card__buy--heart', {
+            'card__buy--heart-active': isFavorite,
+          })}
+          onClick={handleFavButton}
+        >
```

Comment:
We need to create an array where all liked items will be added and then displayed on Favourites page. At the moment when you click on like button it moves you to Favourites page which is not the behaviour we want. You can check it out on our [demo](https://fe-sep22-hope-it-works.github.io/frontend-phone-catalog-app) link.